templates/repo/wiki/view.tmpl

```diff
@@ -85,7 +85,7 @@
 			{{end}}
 		</div>
 		{{if .footerPresent}}
-		<div class="ui grey segment">
+		<div class="ui segment">
```

Comment:
why remove`grey`?

Reply:
Because the footer was the only one with a grey line on the top; neither the main wiki page nor the sidebar had one, so I removed it for consistency.